cpp/src/community/louvain.cuh

```diff
@@ -30,6 +30,41 @@
 
 namespace cugraph {
 
+template <typename T>
+inline void print_v(const char *label, rmm::device_vector<T> const &vector_v)
```

Comment:
You can use

https://github.com/rapidsai/raft/blob/branch-0.16/cpp/include/raft/cudart_utils.h#L249
```
template <class T, class OutStream>
void print_device_vector(const char* variable_name, const T* devMem,
                         size_t componentsCount, OutStream& out)
```
and print_host_vector as well.

Reply:
These are solely for debugging purposes.  They will be deleted before the final version is used.